src/test/scala/com/sksamuel/scapegoat/inspections/equality/ComparingUnrelatedTypesTest.scala

```diff
@@ -56,10 +56,10 @@ class ComparingUnrelatedTypesTest extends InspectionTest {
       }
       "for char" - {
         "compared to char-sized long literal" in {
-          verifyNoWarnings("""object A { val c = 'a'; val c = l == 97L }""")
+          verifyNoWarnings("""object A { val c = 'a'; val l = c == 97L }""")
```

Comment:
This test was throwing and error but not failing because it was no scapegoat warning.